packages/helium-cli/src/submit-price.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  init,
+} from "@helium/price-oracle-sdk";
+import * as anchor from "@coral-xyz/anchor";
+import {
+  PublicKey,
+} from "@solana/web3.js";
+import os from "os";
+import yargs from "yargs/yargs";
+import { BN } from "bn.js";
+import axios from 'axios';
+
+const { hideBin } = require("yargs/helpers");
+
+async function findBinancePrice(symbol: string): Promise<number> {
+  try {
+    const response = await axios.get(`https://api.binance.com/api/v3/ticker/price?symbol=${symbol}`);
+    const data = response.data;
+    return parseFloat(data.price);
+  } catch (error) {
+    console.error(`Error fetching ticker price for ${symbol}: ${error.message}`);
+    throw error;
+  }
+}
+
+async function run() {
+  const yarg = yargs(hideBin(process.argv)).options({
+    wallet: {
+      alias: "k",
+      describe: "Anchor wallet keypair",
+      default: `${os.homedir()}/.config/solana/id.json`,
+    },
+    url: {
+      alias: "u",
+      default: "http://127.0.0.1:8899",
+      describe: "The solana url",
+    },
+    price: {
+      type: "number",
+      required: false,
+      describe: "The price to submit. Required if not using default pricing"
+    },
+    priceOracle: {
+      type: "string",
+      required: true,
+      describe: "Address of the price oracle to submit a price to"
+    },
+    useBinancePrice: {
```

Comment:
ideally we add a useCoingeckoPrice as well. Current implementation: https://github.com/helium/helium-wallet-rs/blob/a7e9c0721e6ab6511b928745aed0564e1ffe4f6c/src/cmd/oracle.rs

Reply:
Sounds good. Bilaxy too?

Reply:
negative. Just Coingecko